src/java.base/share/classes/java/io/BufferedInputStream.java

```diff
@@ -50,6 +50,12 @@
  * reread before new bytes are  taken from
  * the contained input stream.
  *
+ * <p> In general, different {@code BufferedInputStream}s should not be used
+ * to read from the same {@code InputStream}. There is no way for a
+ * {@code BufferedInputStream} to know what another {@code BufferedInputStream}
+ * has read from the underlying {@code InputStream}, nor dues it have access
```

Comment:
Hello Brian, there's a typo here - should have been "does" instead of "dues".

Reply:
Fixed in [9c02419](https://github.com/openjdk/jdk/pull/20320/commits/9c02419241f31ff5c2c665eb3d934599f216445f), thanks.